content/applications/finance/accounting/bank/loans.rst

```diff
@@ -0,0 +1,72 @@
+:show-content:
```

Comment:
```suggestion
```
This metadata markup is only necessary if you have other rst pages under a page (not the case here)